playbooks/uda-workflow/connector-install/install_sap_hana_plugin.yml

```diff
@@ -0,0 +1,68 @@
+# => Cohesity Agent Management
+# =>
+# => Collection: cohesity.dataprotect
+# =>
+
+# => Install the Cohesity Datastore Agent on each SapHana host
+# => specified in the Ansible inventory
+# =>
+---
+- hosts: saphana
+  # => We need to gather facts to determine the OS type of
+  # => the machine
+  gather_facts: true
+  vars:
+    state: "present"
+    platform: SapHana
+    saphana_admin: sssadm
+    saphana_password: fr8shst8rt
+    saphana_home: /usr/sap/SSS/home
+    SID: SSS
```

Comment:
What is this used for ?

Reply:
This and user_store_key ? 

Are these part of cohesity documentation ?

Reply:
yeah, these are required parameter for the connector plugin installation.